docs/design/design_v1.9.md

```diff
@@ -69,196 +67,246 @@ This means we aim to standardize:
- - `--allow-privileged` to `true` 
-
 
-#### Controller Manager
+#### Controller manager
 
 The controller-manager needs to know this in particular:
- - The Pod Network CIDR if any; also enables the Subnet Manager feature required for some CNI network plugins (flag  `cluster-cidr`, `node-cidr-mask-size` and `allocate-node-cidrs=true` respectively)
+- If kubeadm is invoked specifying a `--pod-network-cidr`, the Subnet manager feature required for some CNI network plugins is enabled by 
+   setting `--allocate-node-cidrs=true`,  `--cluster-cidr` and `--node-cidr-mask-size` flags are set.
+ - If a cloud provider is specified, the corresponding `--cloud-provider` is specified, together with  the  `--cloud-config` path if such configuration file exists.
 
 Other flags that are set unconditionally:
- - The `BootstrapSigner` and `TokenCleaner` controllers are enabled
- - `--root-ca-file` to `ca.crt`
+ - Enables all the default controllers plus `BootstrapSigner` and `TokenCleaner` controllers for TLS bootstrap.
+ - `--root-ca-file` to `ca.crt`.
  - `--cluster-signing-cert-file` to `ca.crt`, if External CA mode is disabled, otherwise to `""`.
  - `--cluster-signing-key-file` to `ca.key`, if External CA mode is disabled, otherwise to `""`. 
- - `--service-account-private-key-file` to `sa.key`
- - `--use-service-account-credentials` to `true`
+ - `--service-account-private-key-file` to `sa.key`.
+ - `--use-service-account-credentials` to `true`.
 
 #### Scheduler
 
-kubeadm doesn't set any special scheduler flags.
+Kubeadm doesn't set any special scheduler flags.
+
+### Generate static Pod manifest for local etcd
+
+If the user specified an external etcd this step will be skipped, otherwise a static manifest file will be generated for creating a local etcd instance running in a Pod with following attributes:
+
+- listen on `localhost:2379` and use `HostNetwork=true`.
+- make a `hostPath` mount out from the `dataDir` to the host's filesystem.
+- Any extra flags specified by the user.
+
+Please note that:
+
+1. The etcd image will be pulled from `gcr.io/google_containers`. In case an alternative image repository is specified this one will be used; In case an alternative image name is specified, this one will be used.
+2. in case of `kubeadm`  executed in the `--dry-run` mode, the etcd static Pod manifest is written in a temporary folder.
+3. Static Pod manifest generation for local etcd can be invoked individually with the `kubeadm alpha phase etcd local` command.
 
-### Generate Static Pod Manifest for local etcd
+### (optional) Write init kubelet configuration
 
-If the user specified an external etcd this step will be skipped, otherwise a static manifest file will be generated for creating a local etcd instance running in a pod with following attributes:
+If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig`, it writes the kubelet init configuration into `/var/lib/kubelet/config/init/kubelet` file.
```

Comment:
kubelet init configuration (can be changed in the using the Config File under `.kubeletConfiguration.baseConfig`) into the `/var/lib/kubelet/config/init/kubelet` file.